### PR TITLE
tools/functional-tester: add http status reporter

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

**Comment by xiang90** [comment]
@yichengq PTAL

File: tools/functional-tester/etcd-tester/main.go [hunk 1]
```diff
@@ -43,5 +44,9 @@ func main() {
 		cluster: c,
 		limit:   *limit,
 	}
-	t.runLoop()
+	go t.runLoop()
```
Comment (yichengq) [inline tools/functional-tester/etcd-tester/main.go]: the number of loop may be limited, and it should exit after finish.

File: tools/functional-tester/etcd-tester/main.go [hunk 2]
```diff
@@ -43,5 +44,12 @@ func main() {
 		cluster: c,
 		limit:   *limit,
 	}
+
+	sh := statusHandler{status: &t.status}
+	http.Handle("/status", sh)
+	go func() {
+		log.Fatal(http.ListenAndServe(":9028", nil))
+	}()
```
Comment (yichengq) [inline tools/functional-tester/etcd-tester/main.go]: one line.

**Comment by yichengq** [comment]
lgtm

**Pull request merged**